source/speech/__init__.py

```diff
@@ -230,10 +230,64 @@ def speakSpelling(
 	speak(seq, priority=priority)
 
 
-# C901 'getSpellingSpeech' is too complex
-# Note: when working on getSpellingSpeech, look for opportunities to simplify
-# and move logic out into smaller helper functions.
-def getSpellingSpeech(  # noqa: C901
+def getSpellingSpeechAddCharMode(

```

Comment:
Please make this a private method for now:
```suggestion
def _getSpellingSpeechAddCharMode(

```